index.php

```diff
@@ -7,8 +7,28 @@
 
       for ($i=0; $i < $length ; $i++) {
         if ($datasetNames[$i] !== '.' && $datasetNames[$i] !== '..') { //don't need link for directory . and ..
-          echo '<a target="_blank" href="../graph.php?ds_id=' . $datasetNames[$i] . '">' . $datasetNames[$i] . '</a>';
-          echo '<br>';
+
+          //following are the file names that we expect
+          $metadataFile = $datasetNames[$i] . '_metadata.tsv';
```

Comment:
This is fine.

But from a long term view, maybe we could bundle this up into a function so that we could easily move it to another php application in the future.